groups/groups_test.go

```diff
@@ -115,6 +115,33 @@ func TestStagingEmailLength(t *testing.T) {
 	}
 }
 
+// TestDescriptionLength tests that the number of characters in the
+// google groups description does not exceed 300.
+//
+// This validation is needed because gcloud allows apps:description
+// with length no greater than 300
+func TestDescriptionLength(t *testing.T) {
+	var errs []error
+	for _, g := range cfg.Groups {
+		description := g.Description
+
+		len := utf8.RuneCountInString(description)
+		//Ignoring this group's description for backwards compatibility because it seems at
+		//the time of that group's creation, this description limit was not imposed
+		if len > 300 && g.Name != "k8s-infra-staging-cip-test" {
```

Comment:
I would rather we truncate this group's description and avoid the special case

Looking at https://github.com/kubernetes/k8s.io/blob/a7d10e5664779ec6c47be2b91db64f00e496617b/groups/sig-release/groups.yaml

I would suggest changing the description to:
```
ACL for staging cip-test buckets, the test-only backup, and auditing
jobs for the promoter. See infra/gcp/ensure-prod-storage.sh for more
details.
```

Reply:
This should be fixed
```
pjoglekar@pjoglekar-a01 groups % make test
go test
PASS
ok  	k8s.io/k8s.io/groups	0.229s

```